sympy/vector/tests/test_vector.py

```diff
@@ -3,9 +3,9 @@
      sin, cos, Function, Integral, Derivative, diff, integrate
 from sympy.vector.vector import Vector, BaseVector, VectorAdd, \
      VectorMul, VectorZero
-from sympy.vector.coordsysrect import CoordSysCartesian
+from sympy.vector.coordsysrect import CoordSys3D
 
-C = CoordSysCartesian('C')
+C = CoordSys3D('C')
```

Comment:
You have to filter them so that the travis tests don't error but simply emit the warning.